modules/openapi-generator-gradle-plugin/src/main/kotlin/org/openapitools/generator/gradle/plugin/OpenApiGeneratorPlugin.kt

```diff
@@ -138,6 +138,7 @@ class OpenApiGeneratorPlugin : Plugin<Project> {
                     generateModelDocumentation.set(generate.generateModelDocumentation)
                     generateApiTests.set(generate.generateApiTests)
                     generateApiDocumentation.set(generate.generateApiDocumentation)
+                    splitOperation.set(generate.splitOperation)
```

Comment:
Do we need to make similar change in the CLI as well?

Reply:
@wing328 you talk about this repo https://github.com/OpenAPITools/openapi-generator-cli ?

Reply:
will check